src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -313,10 +315,6 @@ def _encode_prompt(
             batch_size = prompt_embeds.shape[0]
 
         if prompt_embeds is None:
-            # textual inversion: procecss multi-vector tokens if necessary
```

Comment:
Why delete this?

Reply:
> Why delete this?
Sorry for that. I think I deleted it by mistake when I used black to format the code.